src/Web.DataAccess/Db/Commands/BaseDbCommand.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Byndyusoft.Dotnet.Core.Samples.Web.DataAccess.Db.Commands
+{
+    using System;
+    using Infrastructure.Dapper;
+    using Infrastructure.Dapper.SessionsFactory;
+    using Microsoft.Extensions.Options;
+
+    public abstract class BaseDbCommand
+    {
+        protected readonly ISessionsFactory SessionsFactory;
+        protected readonly SqlExecutionOptions SqlExecutionOptions;
+
+        protected BaseDbCommand(ISessionsFactory sessionsFactory, IOptions<SqlExecutionOptions> sqlExecutionOptions)
+        {
+            if (sqlExecutionOptions == null)
+                throw new ArgumentNullException(nameof(sqlExecutionOptions));
+
+
+            SessionsFactory = sessionsFactory ?? throw new ArgumentNullException(nameof(sessionsFactory));
+            SqlExecutionOptions = sqlExecutionOptions.Value;
```

Comment:
IMHO, caching IOptions<T>.Value in constructor is not a good idea, because the underlying options provider's data may change during program execution. The Extensions.Options infrastructure has an ability to reload values insensibly to program and new all calls to IOptions<T>.Value will return new values. But if you cache value in constructor you will use old one.